### ISSUE TITLE
Automate the application of ethtool parameters to capture interfaces.

### ISSUE THREAD

**Issue by arka0m** [arkime/arkime#1729]
**Is your feature request related to a problem? Please describe.**
When adding or modifying a capture interface, editing the data/moloch/etc/config.ini file is not sufficient. (https://arkime.com/faq#how-do-i-monitor-multiple-interfaces)
In some cases, this may cause the molochcapture service to crash.

Indeed, after modifying an interface in /data/moloch/etc/config.ini, when I start the molochcapture service, the ETHTOOL parameters try to apply in a loop and the molochcapture service fails to start.
![molochcapture-start](https://user-images.githubusercontent.com/63341497/128087014-5b56427c-7554-4302-84b3-48a08fcb33f3.PNG)
![molochcapture-start2](https://user-images.githubusercontent.com/63341497/128087040-770729d9-e7d2-40a7-a332-f428106389cc.PNG)

While looking for a solution, I could see that a pre-script (/data/moloch/bin/moloch_config_interfaces.sh) is launched. This script is applied to the interfaces initially filled in. It is therefore necessary to add/modify the new interface in this file for the service to work properly.
After a simple addition of a network interface, the service starts correctly, but the new interface does not have the right settings for capture. Whereas, after an interface modification, it can make the service crash.

**Describe the solution you'd like**
Automatically apply the right parameters to the right interfaces when starting the molochcapture service.

**Describe alternatives you've considered**
A quick way to solve this problem seems to be to include this file for modification in the documentation. However, modifying the pre-script to automate this would make the user experience more pleasant, easier and more complete.
For my own purposes, I have improved the original pre-script. Maybe this little contribution will help or be useful to someone.

The following print screen shows the call to the modified script with the config.ini file as argument :
![molochcapture_modified-service](https://user-images.githubusercontent.com/63341497/128087596-a868eaf6-3f74-4997-b1d3-b5470119295c.png)

The code below corresponds to the modified pre-script :
![image](https://user-images.githubusercontent.com/63341497/128088136-be32e18b-ba54-4d1e-b859-3173497996ab.png)

**Additional context**
During my search for a solution and attempted crash replay, I identified another source of the molochcapture service not working.
If an interface is not used, the molochcapture service crashes the same way. Wouldn't it be possible to start the default interface in the pre-script: moloch_config_interfaces.sh? Even if it is obvious that an interface has to be up and running before starting the service, start them by default in the pre-script might be a good solution.

**Comment by awick** [comment]
Good idea, would also need to support passing in the node name and using the interface in there if there is one before using the default one. Also class isn't always set. Could you submit a PR with that solution?

**Comment by arka0m** [comment]
Of course, I implement the use of the node and consolidate the different use cases. Then I will do a PR.
